tasks/systemd.yml

```diff
@@ -0,0 +1,42 @@
+---
+# receives a structure like
+# unit_item
```

Comment:
perhaps "unit_items" since technically more than one can be passed?

Reply:
The systemd tasks is actually receiving only one unit at a time, please look at the `tasks/main.yml` file.

Reply:
Actually, based on the block in `tasks/main`, I believe a list is being passed with the *include* as is (see the current implementation which has a similar construct). Thinking you already have but try placing a `debug` at the beginning of the tasks file to investigate further.

Anyway, thinking I'd save the file restructure for another commit and go with the action/activation update.

Reply:
Hmmm..maybe not. afk atm but will double-check in bit.

Reply:
Any further comments?